javascript_nodejs/14.nlp-with-dispatch/dialogs/homeAutomation/home-automation-state.js

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const HOME_ATUOMATION_STATE_PROPERTY = 'homeAutomation.state';
+
+class DeviceState {
+    /**
+     * 
+     * @param {String} device 
+     * @param {String} room 
+     * @param {String} deviceProperty 
+     * @param {String} deviceState 
+     */
+    constructor(device, room, deviceProperty, deviceState) { 
+        this.deviceName = device?device:'';
+        this.room = room?room:'';
+        this.deviceProperty = deviceProperty?deviceProperty:'';
+        this.deviceState = deviceState?deviceState:'';
```

Comment:
whitespace comments applies to all here.

Reply:
Fixed.